src/components/AudioDetail.vue

```diff
@@ -60,6 +60,14 @@
                     @input="
                       changePreviewAccent(accentPhraseIndex, parseInt($event))
                     "
+                    @wheel="
+                      changeAccentByScroll(
+                        accentPhraseIndex,
+                        accentPhrase.moras.length,
+                        accentPhrase.accent,
+                        parseInt($event.deltaY / 100)
```

Comment:
In my computer environment, deltaY is 200, so the slider has moved two steps.
How can we align to one step in any environment?